content/sql/concepts/window-functions/terms/row_number/row-number.md

```diff
@@ -0,0 +1,132 @@
+* **`ORDER BY`** defines the order within each partition. It is a mandatory parameter, because **`ROW_NUMBER()`** is order sensitive.
+* **`ROW_NUMBER()`** can also be used for **pagination**, dividing the data set into discrete increments. For example, if a result set would have 100 rows, adding a **`WHERE`** clause would allow the user to selectively view only results 11-20.
+
+## Example
+This function is useful when creating tables. The following example uses a dataset, **`customers`**, with entries for **`first_name`** , **`last_name`** , and **`city`**(where they live).
+
+| first_name | last_name | city         |
+| ---        | ---       | ---          |
+| Sarah      | Myer      | Houston      |
+| Susan      | Davidson  | Dallas       |
+| Mary       | Greene    | Raleigh      |
+| Joseph     | Chang     | Raleigh      |
+| Eric       | Gustav    | Louisville   |
+| Chris      | Blake     | Manchester   |
+| Tyler      | Hunter    | Houston      |
+| Matthew    | Rivera    | Louisville   |
+| Samantha   | Daniels   | Dallas       |
+| Emily      | Pugh      | Manchester   |
+
+Using the following statement, the above information is pulled from the dataset **`customers`**. It is ordered alphabetically by their first name, then each row is assigned an integer starting with '1'.
+
+```sql
+SELECT 
+   ROW_NUMBER() OVER (
+	ORDER BY first_name
+   ) row_num,
+   first_name, 
+   last_name, 
+   city
+FROM 
+   customers;
+```
+This creates the following output. Notice that the above statement does not have a **`PARTITION BY`** parameter. This essentially creates just one partition, the result set.
+
+| row_num | first_name | last_name | city        |
+| ---     | ---        | ---       | ---         |
+| 1       | Chris      | Blake     | Manchester  |
+| 2       | Emily      | Pugh      | Manchester  |
+| 3       | Eric       | Gustav    | Louisville  |
+| 4       | Joseph     | Chang     | Raleigh     |
+| 5       | Mary       | Greene    | Raleigh     |
+| 6       | Matthew    | Rivera    | Louisville  |
+| 7       | Samantha   | Daniels   | Dallas      |
+| 8       | Sarah      | Myer      | Houston     |
+| 9       | Susan      | Davidson  | Dallas      |
+| 10      | Tyler      | Hunter    | Houston     |
+
+For **pagination**, the function would be changed like so to only show the first 5 entries:
+
+```sql
+WITH customers AS (
+    SELECT 
+        ROW_NUMBER() OVER (
+             ORDER BY 
+                first_name
+        ) row_num, 
+        first_name, 
+        last_name,
+        city
+     FROM 
+        customers
+) SELECT 
+    first_name, 
+    last_name,
+    city
+FROM 
+    customers
+WHERE 
+    row_num > 0 AND 
+    row_num <= 5;
+```
+
```

Comment:
Can we have a section here showing the results in a table like the others?


Chris|Blake|Manchester
Emily|Pugh|Manchester
Eric|Gustav|Louisville
Joseph|Chang|Raleigh
Mary|Greene|Raleigh